src/messagingsession/MessagingSessionConfiguration.ts

```diff
@@ -30,13 +30,17 @@ export default class MessagingSessionConfiguration {
   /**
    * Constructs a MessagingSessionConfiguration optionally with userArn, messaging session id, a messaging session
    * endpoint URL, and the chimeClient.
+   *
+   * endpointUrl is deprecated as internally it resolved on connect via chimeClient if undefined, and always re-resolved
```

Comment:
nit: "..it is resolved on connect..". Also could you please end this with a ".".